tailwind.base.css

```diff
@@ -2,6 +2,10 @@
 @tailwind components;
 @tailwind utilities;
 
+:root {
+  --gn-ui-text-input-background-color: #f0f0f0;
+}
```

Comment:
Where does this get applied? Shouldn't we rather move it to `.gn-ui-text-input` (if it's necessary)?

Reply:
I forgot to remove it.